### ISSUE TITLE
No video output after upgrading to Mojave

### ISSUE THREAD

**Issue by romancin** [sakoula/hackintosh.hp.800.g2#2]
Hi! Thank you for this guide, it has been working perfect till High Sierra 10.13.6. Yesterday I upgraded to Mojave and lost video output. Every other device is working (I had vnc configured and I can connect without problem). Do you know how to fix? Thank you.

**Comment by sakoula** [comment]
I have not tried to upgrade yet. I will not probably upgrade for a couple of months. I will update the guide with some more changes I have made. 

Are you using two monitors? 

Try removing all the properties in config.plist > Devices > IntelGFX properties. and use Lilu.kext+IntelGraphicsFixup.kext as described in https://hackintosher.com/guides/updating-your-hackintosh-to-mojave-10-14/

Let me know if this works.

Also send me some debug info using https://github.com/black-dragon74/OSX-Debug

**Comment by romancin** [comment]
Thank you for your answer sakoula. After reading a lot around this problem, I have changed to Lilu + WhatEverGreen and changing config.plist accordingly, as stated in this link: https://www.tonymacx86.com/threads/guide-intel-framebuffer-patching-using-whatevergreen.256490/

Everything is working now :)

**Comment by sakoula** [comment]
Hi romancin! 
Glad to know that it works now!!!
Can you share with me your config.plist (remember to anonymize the smbios part) to check it out before I upgrade? Thanks!

**Comment by romancin** [comment]
Of course, here you have it.


[config.plist.zip](https://github.com/sakoula/hackintosh.hp.800.g2/files/2418306/config.plist.zip)

For reference, this is the Other folder in Clover kexts
![captura de pantalla 2018-09-26 a las 8 16 40](https://user-images.githubusercontent.com/17579835/46060708-83631800-c164-11e8-8f3a-47a528090be0.png)

**Comment by sakoula** [comment]
Thanks!

BTW I have managed to have the audio work with AppleALC (had to make a change on the existing patch). The patch is already in the official repository.

In order to make it work you have to:

1. delete VoodooHDA.kext from your system
2. config.plist > Devices > Audio Inject 11
3. config.plist > ACPI > DSDT > Patches > Rename HDAS to HDEF (48444153 - 48444546)
4. install https://github.com/acidanthera/AppleALC (requires Lilu too but your already have it)
5. you should have native audio!

**Comment by romancin** [comment]
This is for the Internal Speaker? HDMI Audio works well without changing anything.

**Comment by sakoula** [comment]
yes it is for the internal speaker as well. Voodoo is not the right way for enabling audio. This patch will give you better quality. I have not tried HDMI audio. Out of curiosity how do you use hdmi audio? What kind of monitor are you using and how do you have it connected?

**Comment by romancin** [comment]
I use it for media purposes connected to the TV only, Kodi , OpenEmu etc, but I will try your indications in order to see if sound is better.
Could you please paste here config.plist changes for the Audio? Thank you!

**Comment by sakoula** [comment]
add in ACPI > DSDT > Patches:

```
			<key>Patches</key>
			<array>
				<dict>
					<key>Comment</key>
					<string>Rename HDAS to HDEF</string>
					<key>Disabled</key>
					<false/>
					<key>Find</key>
					<data>
					SERBUw==
					</data>
					<key>Replace</key>
					<data>
					SERFRg==
					</data>
				</dict>
			</array>
```

change in Devices > Audio:

```
		<key>Audio</key>
		<dict>
			<key>Inject</key>
			<integer>11</integer>
			<key>ResetHDA</key>
			<true/>
		</dict>
```

and install the Lilu + ApplALC after removing Voodoo

BTW the machine does not have a HDMI port, how do you connect it to the TV? Via the DP port?

**Comment by romancin** [comment]
Thank you!

Yes, I use a DP to HDMI Adapter for it. Working flawlessly and Audio passthrough too.
